assessment_software/src/app/table-component/table.component.html

```diff
@@ -0,0 +1,14 @@
+<table>
+    <tr>
+        <th>Name</th>
+        <th>ID</th>
+        <th>Passcode</th>
+        <th>Email</th>
+    </tr>
+    <tr *ngFor="let d of data">
+        <td [width]="width">{{d.name}}</td>
+        <td [width]="width">{{d.ID}}</td>
+        <td [width]="width">{{d.passcode}}</td>
+        <td [width]="width">{{d.email}}</td>
+    </tr>
+</table>
```

Comment:
It is better to have a dynamic table since we have two tables. currently, the header is statically defined as Name, ID, Passcode, and Email. But, it is best to take the list of headers from the user. From the header, you can get the number of columns. (this is going to be interesting implementation since you will have to do one ngFor inside another like nested for loop)